core/overlays/stage/common/configmaps.yaml

```diff
@@ -37,6 +37,10 @@ metadata:
 data:
   solvers: |
     solver-rhel-8-py36
+    solver-fedora-31-py38
+    solver-fedora-31-py37
```

Comment:
Do we want to invest time to Fedora 31? I think we could include F33 instead.

Reply:
https://endoflife.software/operating-systems/linux/fedora

Reply:
ack, will remove the f31.